views/partials/release-ticket-diff-add.jade

```diff
@@ -9,12 +9,12 @@ div(class='modal-body')
     ng-submit='submit(addDiff.$valid)'
   )
     div(class='form-group')
-      label(class='control-label col-sm-2') Phabricator Diff
+      label(class='control-label col-sm-2') Phabricator Diff/Audit
```

Comment:
I don't think it's necessarily a link to an "Audit". It's a link to a "Commit", sometimes those commits require Audit.
(shrug)

Reply:
That is indeed the case and, depending on which repo is being committed to, the audit may not be created immediately. "Commit" would be more appropriate terminology.

Reply:
Done.